test/domain.test.ts

```diff
@@ -13,6 +14,44 @@ describe('store: [adapter=DOMAIN]', () => {
     Object.assign(process, { domain: undefined });
   });
 
+  describe('initialize()', () => {
+    it('should initialize the store.', done => {
+      const callback = () => {
+        expect(!!(process.domain as any)[STORE_KEY]).to.equal(true);
+        expect(globalStore.isInitialized()).to.equal(true);
+        done();
+      };
+
+      globalStore.initialize(adapter)(callback);
+    });
+
+    it('should also bind params if params are passed through arguments.', done => {
+      const req = new EventEmitter();
+      const res = new EventEmitter();
+      const emitters = [req, res];
+      const errorCallback = (err: any) => err;
+
+      const callback = () => {
+        expect(!!(process.domain as any)[STORE_KEY]).to.equal(true);
+        expect(globalStore.isInitialized()).to.equal(true);
```

Comment:
This part of the assertion is already in `it('should initialize the store.', ...`. I don't think we need to do the same here. If the first test passes, this should pass as well.